src/components/Chart/Display.svelte

```diff
@@ -17,6 +19,12 @@
 
     let canvas: HTMLCanvasElement;
 
+    let chart: Chart;
+
+    // Granularity is in the form of seconds. If left undefined,
+    // backend uses the predetermined quantum as the selected granularity
+    export let granularity: Granularity | undefined = undefined;
```

Comment:
Assigning `undefined` is redundant and contradictory.
```suggestion
    export let granularity: Granularity | undefined;
```

Reply:
Not putting a default prop value of `undefined` raises a diagnostic that prop is possibly undefined on usage although typed as `T | undefined`.

Reply:
[3762139](https://github.com/drippy-iot/dashboard/pull/20/commits/3762139755e7aa38bfc28efb525b4d5d7ae31819) adds `Granularity.REALTIME` as its default value instead.